brave/ui/app/store/actions.js

```diff
@@ -4,7 +4,17 @@ MetaMaskActions.addToken = addToken
 MetaMaskActions.setBatTokenAdded = setBatTokenAdded
 MetaMaskActions.SET_BAT_TOKEN_ADDED = 'SET_BAT_TOKEN_ADDED'
 
+MetaMaskActions.setRewardsDisclosureAccepted = setRewardsDisclosureAccepted
+MetaMaskActions.REWARDS_DISCLOSURE_ACCEPTED = 'REWARDS_DISCLOSURE_ACCEPTED'
```

Comment:
as above, not sure we need this if we just use `actions.setRewardsDisclosureAccepted`.

Reply:
done